### PR TITLE
Add proper Stetho debugging setup documentation

### PR DESCRIPTION
**Pull request by louisgv**
### Explain the **motivation** for making this change. What existing problem does the pull request solve?

Adding the proper way to setup Stetho debugging, based on this post: https://medium.com/@andr3wjack/stetho-with-react-native-87642e5d7131#.352jqqavc

with updated OkHttp API.

**Test plan**

 + Manual test, DYI.

### CODE REVIEW

File: docs/Debugging.md [hunk 1]
```diff
@@ -96,43 +96,87 @@ The debugger will receive a list of all project roots, separated by a space. For
...
+    ```java
+    public class StethoWrapper {
+
+        public static void initialize(Context context) {
+            // NO_OP
+        }
+
+        public static void addInterceptor(OkHttpClient okHttpClient) {
+            // NO_OP
+        }
```
Comment (AndrewJack) [inline docs/Debugging.md]: Can change to
```java
public static void addInterceptor() {
   // NO_OP
}
```

File: docs/Debugging.md [hunk 2]
```diff
@@ -96,43 +96,87 @@ The debugger will receive a list of all project roots, separated by a space. For
...
+      
+          OkHttpClient client = new OkHttpClient.Builder()
+          .connectTimeout(0, TimeUnit.MILLISECONDS)
+          .readTimeout(0, TimeUnit.MILLISECONDS)
+          .writeTimeout(0, TimeUnit.MILLISECONDS)
+          .cookieJar(new ReactCookieJarContainer())
+          .addNetworkInterceptor(new StethoInterceptor())
+          .build();
+      
+          StethoWrapper.addInterceptor(client);
```
Comment (AndrewJack) [inline docs/Debugging.md]: replace line 160 to 186 with:
```java
StethoWrapper.addInterceptor();
```

File: docs/Debugging.md [hunk 3]
```diff
@@ -96,43 +96,87 @@ The debugger will receive a list of all project roots, separated by a space. For
 
 ### Debugging with [Stetho](http://facebook.github.io/stetho/) on Android
 
+(Adapted from https://medium.com/@andr3wjack/stetho-with-react-native-87642e5d7131#.352jqqavc)
```
Comment (AndrewJack) [inline docs/Debugging.md]: Don't worry about adding this. Blogs get out of date, hopefully docs don't 😄

File: docs/Debugging.md [hunk 4]
```diff
@@ -96,43 +96,83 @@ The debugger will receive a list of all project roots, separated by a space. For
...
-   import com.facebook.stetho.Stetho;
-   import okhttp3.OkHttpClient;
-   import com.facebook.react.modules.network.OkHttpClientProvider;
-   import com.facebook.stetho.okhttp3.StethoInterceptor;
-   import java.util.concurrent.TimeUnit;
+Go to http://facebook.github.io/stetho/ and replace 1.x.x with the latest version. For example:
+   
+   ```gradle
+    debugCompile 'com.facebook.stetho:stetho:1.4.2'
+    debugCompile 'com.facebook.stetho:stetho-okhttp3:1.4.2'
```
Comment (AndrewJack) [inline docs/Debugging.md]: Could remove this bit and specify the version to use.
The version is dependant on the version of OkHttp React native is compatible is.

Comment (hramos) [inline docs/Debugging.md]: I updated this PR prior to seeing this comment. Would you prefer we recommend a specific version that we know is compatible with RN, or is there a process we can document for people to figure out what version of OkHttp they should be using here?

**Comment by hramos** [comment]
Waiting for some clarification on OkHttp

**Pull request closed without merging**